solvebio/resource/vault.py

```diff
@@ -0,0 +1,70 @@
+"""Solvebio Vault API resource"""
+# from ..client import client
+from ..help import open_help
+
+from .apiresource import CreateableAPIResource
+from .apiresource import ListableAPIResource
+from .apiresource import SearchableAPIResource
+from .apiresource import UpdateableAPIResource
+from .apiresource import DeletableAPIResource
+
+
+class Vault(CreateableAPIResource,
```

Comment:
To get an object by its ID, you must use the Object class:

```
Object.retrieve(488353213969592764)
```

To get an object by its path, again you must use the Object class:

```
Object.all(path='/1000G')
Object.all(path='/1000G', vault_id=7205)
Object.all(path='/1000G', vault_name='public')
```

Search functionality in the API is presently built on top of the Logical Object model.  Accordingly, searching within the client can be performed by the correct `Object.all(query=xxx)` call:

```
Object.all(query='GRC', vault_id=5906)
```